.env

```diff
@@ -106,16 +106,16 @@ NEXT_PUBLIC_L1_PUFFER_GATEWAY_PROXY_ADDR = ""
 NEXT_PUBLIC_L2_PUFFER_GATEWAY_PROXY_ADDR = ""
 NEXT_PUBLIC_L1_STANDARD_ERC20_GATEWAY_PROXY_ADDR = ""
 NEXT_PUBLIC_L2_STANDARD_ERC20_GATEWAY_PROXY_ADDR = ""
-NEXT_PUBLIC_L1_SCROLL_MESSENGER = "0xE1117A5DCb94d0e28931AF95A00f5606619d358C"
+NEXT_PUBLIC_L1_SCROLL_MESSENGER = "0x5EF9a083704451F60d76d1aCf59b7d9E4Ae8CAC8"
```

Comment:
I'd say it's not a safe option to push `.env`. I would prefer to add these default options to the [.env.local.template](https://github.com/t1protocol/frontends/blob/canary/.env.local.template).

Reply:
FYI this was a direct suggestion from Orest